WordPress/Classes/Apps/Reader/ReaderTabViewController.swift

```diff
@@ -1,11 +1,30 @@
 import UIKit
 import SwiftUI
+import WordPressUI
 
 final class ReaderTabViewController: UITabBarController {
+    private var menuStore: AnyObject?
+
     override func viewDidLoad() {
         super.viewDidLoad()
 
-        setupViewControllers()
+        // TODO: (reader) remove the need to fetch the menu on first launch before showing anything
```

Comment:
There is currently a known limitation where showing "Recent" and "Discover" requires a topic fetched from the server (in fact, they do not). Until it's refactored, we'll do this once. We'll probably have to keep it until the prototype.